api/views.py

```diff
@@ -91,3 +91,18 @@ def destroy(self, request, *args, **kwargs):
             raise Http404
         Follow.objects.unfollow(followee=followee, follower=follower)
         return Response(status.HTTP_204_NO_CONTENT)
+
+    def retrieve(self, request, *args, **kwargs):
+        followee_id = kwargs['author_pk']
+        follower_id = kwargs['pk']
+        try:
+            followee = get_user_model().objects.get(id=followee_id)
+            follower = get_user_model().objects.get(id=follower_id)
+        except get_user_model().DoesNotExist as e:
+            raise Http404
+
+        try:
+            Follow.objects.get(follower=follower, followee=followee)
+            return Response({'check': True}, status=status.HTTP_200_OK)
+        except Follow.DoesNotExist as e:
+            return Response({'check': False}, status=status.HTTP_200_OK)
```

Comment:
Might be better if we return 404 since the follower could not be found under the followers list, what do you think?